src/site/stages/build/process-cms-exports/transformers/outreach-asset.js

```diff
@@ -0,0 +1,30 @@
+const { get } = require('lodash');
+
+const transform = entity => {
+  const modifiedEntity = {};
+
+  // Derive the raw entity properties.
+  const changed = get(entity, 'changed[0].value');
+  const status = get(entity, 'status[0].value');
+  const title = get(entity, 'title[0].value');
+  const fieldBenefits = get(entity, 'field_benefits[0].value');
+  const fieldDescription = get(entity, 'field_description[0].value');
+  const fieldFormat = get(entity, 'field_format[0].value');
```

Comment:
I actually wasn't sure how to test this function (and thus see what `entity` looked like) so syncing up with @cvalarida since it's not logging from here when running my unit tests! 👍